public/js/actions.js

```diff
@@ -23,5 +36,18 @@ $(document).on('ready', function () {
             default:
                 break;
         }
+        var x = 1;
+        while (x <= game.rowLength * game.colLength) {
+            for (var i = 0; i < game.rowLength; i++) {
+                for (var j = 0; j < game.colLength; j++) {
+                    if (game.board[i][j] === 0) {
+                        $('.data' + x).html(" ")
+                    } else {
+                        $('.data' + x).html(game.board[i][j])
+                    }
+                    x += 1
+                }
+            }
+        }
```

Comment:
and can also be abstracted into it's own function since it kinda seems like you are do the same thing as above.